tezos/2-channel-establishment.md

```diff
@@ -204,6 +203,28 @@ Upon receipt of the `funding_confirmed` message, the merchant:
     - The `context-string` is set to `"zkChannels mutual close"`, as defined in the [global defaults](1-setup.md#Global-defaults). 
   - Checks that the originated contract is confirmed on chain for at least `required_confirmations` blocks.
   - Updates the channel status to `Originated`.
+  
+### The `funding_confirmed` Message
+The customer sends the `funding_confirmed` message to the merchant.
+
+1. type: `funding_confirmed`
+2. data:  none.
+
+
+
+#### Customer Requirements
+
+The customer:
+  
+  - Funds the contract by calling the [`addCustomerFunding` entrypoint](5-tezos-escrow-agent#addCustFunding-entrypoint).
+  - Computes the contract identifier `contract_ID`.
```

Comment:
I think the customer no longer computes this at this stage -- they already got it when they originated.